### ISSUE TITLE
x/website: lighter text reduces contrast/readability

### ISSUE THREAD

**Issue by awly** [golang/go#33014]
New design of golang.org uses lighter color for text (#3e4042) and links (#007d9c).
This makes reading more strenuous on the eyes, especially for navigating stdlib docs.

**Comment by ianlancetaylor** [comment]
CC @andybons @spf13

**Comment by clukawski** [comment]
I agree. I'm looking at the docs quite often, and the new design is rather painful to look at.

**Comment by olegb20** [comment]
Changes for the sake of changes...

The old color scheme worked fine for a decade, everyone got used it. Please bring the old colors back.

**Comment by andybons** [comment]
Please be mindful of https://golang.org/wiki/NoPlusOne

We have the bug marked as needing investigation and are looking into it. Thanks.

**Comment by av86743** [comment]
@andybons 

Do you have a CL opened for this issue?  Thank you.

**Comment by vitali-fedulov** [comment]
The change is not in accordance with the unspoken standard, e.g. the majority of platforms use blue for URLs (Github, Stackoverflow, Wikipedia, Bing, Google etc). Google itself, supposedly came to the font color through many user-based tests. It is probably makes more sense to adjust the new Go logo color than otherwise.

**Comment by seankhliao** [comment]
I believe the text is now darker `#202224`
